### ISSUE TITLE
Vertical Carousel Documentation Example Reproduction not Rounded on Right Side

### ISSUE THREAD

**Issue by Mashiane** [saadeghi/daisyui#1219]
Hi there

I'm trying to reproduce the Vertical Carousel Documentation Example, sadly its not rounded on the right side.

https://play.tailwindcss.com/ozvmrdOTwF

Do I have to add additional classes perhaps to make it work as per docs?

Thanks.

**Comment by NikarashiHatsu** [comment]
I'm pretty sure the issue has something to do with the `width` of the parent.
Using `div.flex` before using the vertical carousel fixed the issue.

Based on your [Tailwind playground](https://play.tailwindcss.com/ozvmrdOTwF):
```html
<div class="flex">
  <div id="car3" class="h-96 carousel carousel-vertical rounded-box">
      <div id="img13" class="carousel-item h-full">
        <img id="img13_image" alt="Test Image"
        src="https://placeimg.com/256/400/arch"></div>
      <div id="img23" class="carousel-item h-full">
        <img id="img23_image" alt="Test Image"
        src="https://placeimg.com/256/400/arch"></div>
      <div id="img33x" class="carousel-item h-full">
        <img id="img33x_image" alt="Test Image"
        src="https://placeimg.com/256/400/arch"></div>
      <div id="img43" class="carousel-item h-full">
        <img id="img43_image" alt="Test Image"
        src="https://placeimg.com/256/400/arch"></div>
      <div id="img53" class="carousel-item h-full">
        <img id="img53_image" alt="Test Image"
        src="https://placeimg.com/256/400/arch"></div>
  </div>
</div>```

**Comment by Mashiane** [comment]
@NikarashiHatsu you are right, the dependency is the parent div. On the DaisyUI website I also noted a flex on the parent div, I removed it and the result was what I added to the playground here. I also changed it from flex to grid and it worked.

All other cases where rounded-box is used, everything works as expected, up until the vertical option is used.

Perhaps let me wait for the lib owner to confirm if this parent dependency is the "real" cause. Thank you so much for looking into it.
